lib/reservation.rb

```diff
@@ -0,0 +1,32 @@
+require 'time'
+require 'date'
+require 'securerandom'
+require_relative 'manager'
+require_relative 'room'
+
+module Hotel
+  class Reservation
+    attr_reader :start_date, :end_date, :rm_num, :cost_per_day, :total_cost, :recloc
+
+    def initialize(start_date:,
+      end_date:, 
+      rm_num: nil, 
+      cost_per_day: 200, 
+      total_cost: nil, 
+      recloc: nil)
```

Comment:
I'm having trouble understanding what this variable represents. It's best to name variables in such a way that their purpose is clear to anyone who might read the code.